compiler_opt/es/blackbox_learner.py

```diff
@@ -275,12 +245,16 @@ def run_step(self, pool: FixedWorkerPool) -> None:
           p for p in initial_perturbations for p in (p, -p)
       ]
 
-    perturbations_as_policies = [
-        self._get_policy_from_perturbation(perturbation)
+    # TODO(boomanaiden154): This should be adding the perturbation to
+    # the existing model weights. That currently results in the model
+    # weights all being NaN, presumably due to rewards not being scaled for
+    # the regalloc_trace problem.
+    perturbations_as_bytes = [
+        perturbation.astype(np.float32).tobytes()
```

Comment:
are they always `np.float32`?

Reply:
Numpy defaults to doubles but the rest of the code expects floats, so they're converted here just to ensure that both sides expect the same width types.